### PR TITLE
chore(visu pages): update visu pages on dev

### PR DESCRIPTION
**Pull request by DumitruCorini**
covers [DEV-272](https://bonitasoft.atlassian.net/browse/DEV-272)

relates to [#225](https://github.com/bonitasoft/bonita-web-pages-sp/pull/225)

[DEV-272]: https://bonitasoft.atlassian.net/browse/DEV-272?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: community/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.bonitasoft.web.application</groupId>
 	<artifactId>bonita-admin-application</artifactId>
-	<version>8.0-SNAPSHOT</version>
+	<version>9.0-SNAPSHOT</version>
```
Comment (rbioteau) [inline community/pom.xml:8]: ?

File: subscription/pom.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	<parent>
 		<groupId>org.bonitasoft.web.application</groupId>
 		<artifactId>bonita-admin-application</artifactId>
-		<version>8.0-SNAPSHOT</version>
+		<version>9.0-SNAPSHOT</version>
```
Comment (rbioteau) [inline subscription/pom.xml:10]: ?

**Review by rbioteau — CHANGES_REQUESTED** [review]

File: community/pom.xml (hunk 1 above)
Comment (DumitruCorini) [reply community/pom.xml:8]: have to rebase this. Was working on the changes

Comment (DumitruCorini) [reply community/pom.xml]: Done

**Review by vhemery — APPROVED** [review]

**Pull request merged**